public/app/plugins/datasource/prometheus/result_transformer.test.ts

```diff
@@ -434,7 +434,7 @@ describe('Prometheus Result Transformer', () => {
                 name: 'Value',
                 type: FieldType.number,
                 values: [30, 10, 40],
-                labels: { le: '3', additionalProperty: '10' },
+                labels: { le: 'Inf+', additionalProperty: '10' },
```

Comment:
oops? :)

Reply:
Haha yes this is a typo, but the `parseSampleValue` should return the same result 🤔 but I guess we're already covering that in another test... I'll fix!